src/components/Editor/EditorPage.tsx

```diff
@@ -93,13 +93,6 @@ export default function EditorPage(props: PageProps): JSX.Element {
           <EditorTopNav />
 
           {typeof window !== 'undefined' && (
-            // <React.Suspense
-            //   fallback={
-            //     <div className="text-center mt-6 font-bold text-2xl">
-            //       Loading
-            //     </div>
-            //   }
-            // >
```

Comment:
what was the reasoning for removing this?

Reply:
I had already commented it out in a previous pr but forgot to completely deleted; it should be useless since all the lazy/suspenseful components it contains already have their own suspense boundaries i think

Reply:
Is there another Suspense boundary for the Editor component? / which suspense boundary is used when the editor is loading?

Reply:
ok never mind it probably doesn't matter as long as there exists a suspense boundary somewhere xD